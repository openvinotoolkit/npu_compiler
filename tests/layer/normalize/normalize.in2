version https://git-lfs.github.com/spec/v1
oid sha256:0bc807bac2d0d5e3c9178a108bcbf4d689a2f1a00681c5ccadad808b25f27104
size 64
