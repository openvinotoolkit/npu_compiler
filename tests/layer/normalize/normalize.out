version https://git-lfs.github.com/spec/v1
oid sha256:66ab30a1fb57bc6d7513cbc6017eed25e850a9fdc64bd1778540334c441f7e11
size 802816
